src/Mod/Path/PathScripts/post/wcsdebug_post.py

```diff
@@ -0,0 +1,283 @@
+# ***************************************************************************
```

Comment:
not sure this file belongs here - it doesn't seem to get shipped which could cause some confusion. Maybe `PathTest` would be a better place for it, or maybe create a dedicated directory for "developer support files".

BTW, I don't think it _should_ get shipped, definitely not in this directory, in case that was an oversight.

PS: I thought there was a way of commenting on a file, didn't find one so this is my poor man's version of that.

Reply:
I put it here because I use symbolic links on my build environment and the post appears in the list of available posts for me. If it's somewhere else, I have to move it to make it available.  But that's just me being selfish. 
It definitely shouldn't ship.  It's just a stripped post to show the names of the ops that are being output.  

Technically I don't need it at all anymore since I moved the buildPostList out, the test cases aren't using it.  I'll remove it.